wcfapp/WcfClient/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.ServiceModel;
+
+namespace WcfClient
+{
+    class Program
+    {
+        static string host;
+
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Client OS: {0}", Environment.OSVersion);
+            host = Environment.GetEnvironmentVariable("host") ?? "localhost";
+            Console.WriteLine("Service Host: {0}", host);
+
```

Comment:
String format variant of WriteLine is overkill. I prefer `"Service Host: " + host` as that gets converted to string.Concat which allocates less and is faster that a string format.

Reply:
Maybe overkill, but I will stay consistent with other occurrences of the similar code in this sample. Good to know though. Thanks 😄.